### DISCUSSION TITLE
Fade out HydrateFallback

### DISCUSSION THREAD

**Discussion by misha-erm** [remix-run/remix#9892] [Q&A]
Hello, 
I'm curious whether it's possible to add exit animation for HydrateFallback. 
In spa app I used to render some splash screen in the root index.html and later add .fade-out in useEffect. But in my Remix app splash screen is needed only for client-loaded routes

Since I don't control how HydrateFallback gets unmounted I'm not sure what's the best approach to achieve that. 
I guess I can put `useEffect` inside HydrateFallback, clone splash screen before unmounting and fade out the clone. Or maybeview transition can be used... But sounds a bit over engineered 

Thank you for any tips

**Comment by terminaate** [comment]
Any updates on this?
